app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -23,11 +23,12 @@
             - else
                 = f.govuk_url_field :application_link, label: { size: "s" }, form_group: { classes: "optional-field" }
           - else
-            = govuk_notification_banner title: t("banners.important") do |banner|
-              - banner.add_heading text: t(".notification.heading")
-              = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
+            - if current_variant?(:"2021_04_application_preview_test", :banner)
+              = govuk_notification_banner title: t("banners.important") do |banner|
+                - banner.add_heading text: t(".notification.heading")
+                = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
             = f.govuk_radio_buttons_fieldset :enable_job_applications do
-              = f.govuk_radio_button :enable_job_applications, true, link_errors: true do
+              = f.govuk_radio_button :enable_job_applications, true, link_errors: true, label: { text: current_variant?(:"2021_04_application_preview_test", :label_link) ? t(".radio_labels.true.label_link.content_html", link: govuk_link_to(t(".radio_labels.true.label_link.link_text"), page_path("job-application-preview"), target: "_blank")) : t(".radio_labels.true.banner") } do
```

Comment:
I'm confused what translation you're using for each of the variants. Shouldn't the label be `Use another application process or form` when the current variant is `:banner`?

Reply:
No, when the banner variant is applied we just want to remove the link from the radio label for 'true':

` radio_labels:
            false: Use another application process or form
            true:
              label_link:
                content_html: Apply using the Teaching Vacancies %{link}
                link_text: application form (opens in new tab)
              banner: Apply through Teaching Vacancies`
              
Perhaps that isn't clear enough?

Reply:
Perhaps that isn't clear enough?

Reply:
Perhaps `banner:` should be `no_label_link:` ?

Reply:
Perhaps. The template and the translation names make it hard to distinguish what's different between the different tests

Reply:
Ok, lemme see what I can do.